core/backend/src/TxnManager.ts

```diff
@@ -47,6 +47,26 @@ export interface TxnChangedEntities {
   readonly updates: EntityIdAndClassIdIterable;
 }
 
+/** @beta */
+export interface QueryLocalChangesArgs {
+  /** Restrict changes to set of classes */
+  readonly includedClasses?: string[];
+  /** Include changes that has not saved yet in addition to saved changes */
+  readonly includeUnsavedChanges?: boolean;
+}
+
+/** Instance key for a change
+* @beta
+*/
+export interface ChangeInstanceKey {
+  /** ECInstanceId of the instance */
+  readonly id: Id64String;
+  /** Class name of the ECInstanceId */
+  readonly classFullName: string;
+  /** Type of change */
+  readonly changeType: "inserted" | "updated" | "deleted";
+}
```

Comment:
```suggestion
/** Represents a change (insertion, deletion, or modification) to a single EC instance made in a local [[BriefcaseDb]].
 * @see [[TxnManager.queryLocalChanges]] to iterate all of the changed instances.
* @beta
*/
export interface ChangeInstanceKey {
  /** ECInstanceId of the instance. */
  id: Id64String;
  /** Fully-qualified class name of the instance. */
  classFullName: string;
  /** The type of change. */
  changeType: "inserted" | "updated" | "deleted";
}
```

Removed `readonly` because unclear why you would restrict callers from assigning to properties of objects you give them ownership of.